.circleci/scripts/terminus-installer.php

```diff
@@ -38,6 +38,8 @@ function downloadTerminus($installdir, $package)
     echo("\nDownloading Terminus " . $version . " from " . $url . " to /tmp \n");
     $couldDownload = file_put_contents("/tmp/" . $package . ".phar", file_get_contents($url));
     echo("Moving to " . $installdir . "...\n");
+    exec("sudo chmod ugo+rwx /tmp/");
+    exec("sudo chmod ugo+rwx /usr/local/bin/");
```

Comment:
I think long-term maintenance would be easier if you removed the Terminus installer, and we instead used the update tool to fetch the Terminus data and create a PR in this repo when things change. It seems to me that it shouldn't be a concern of the docs CI to know what version of Terminus to use, or to install and run it.